### DISCUSSION TITLE
Qt Xmake工程 在VS Code里，头文件找不到

### DISCUSSION THREAD

**Discussion by scially** [xmake-io/xmake#2913] [Q&A] [answered]
在Xmake里写的Qt工程，用的VSCode，Build OK了，可是提示Qt的头文件都找不到
![image](https://user-images.githubusercontent.com/22127608/195245992-07ba1c88-0b84-4370-8da8-1c70530fef3b.png)

**Comment by waruqi** [comment, marked as answer]
那是 intelligense 干的事情，跟 xmake 没啥关系，只要能 build 过，其他跟 xmake 无关。。

intelligense 配置参考 https://github.com/xmake-io/xmake-vscode/issues/40

**Comment by scially** [comment]
可以了。求 增加自动配置 .vscode/compile_commands.json，这个应该是很重要的功能，如果暂时不能自动配置，建议在文档上说明下…… 希望xmake越来越好
